lib/global.d.ts

```diff
@@ -0,0 +1 @@
+declare const __BROWSER_BUILD__: boolean;
```

Comment:
I should've removed this from this branch, but I accidentally did it in the part-2. I'll just leave this as it is. It will be gone when merging part-2 back here.